packages/types/config/render.d.ts

```diff
@@ -7,7 +7,8 @@
  *                https://github.com/jshttp/etag#readme
  */
 
-import { IncomingMessage, ServerResponse } from 'http'
+import { ServerResponse } from 'http'
+import { IncomingMessage } from 'connect'
```

Comment:
While `http` is native library and is fine at first position, it would be preferable to have the `connect` import below `compression` to keep alphabetical order consistency.

Reply:
Done.